src/markerdetector/Pose.java

```diff
@@ -160,14 +160,15 @@ public Pose flipCoords(){
 		translationVector3D.put(0, 0, value1);
 		translationVector3D.put(1, 0, value2);
 		translationVector3D.put(2, 0, value3);*/
-		translationVector3D = MarkerUtils.scalarMultiply(MarkerUtils.matMultiply(r, tvec), -1);
+		translationVector3D = MatMathUtils.scalarMultiply(MatMathUtils.matMultiply(r, tvec), -1);
 		Calib3d.Rodrigues(r, rotationVector3D);
 		return new Pose(rotationVector3D, translationVector3D);
 	}
 
 	/**Returns a hash code for this Pose.
 	@return a hash code for this Pose.
 	*/
+	@Override
```

Comment:
The @Override 'flag' only needs to appear above the sub-class's function and not the super-class's, you should remove these for all the functions in Pose.java

Reply:
Override is necessary for when you override hashCode, toString, and equals because those are object methods

Reply:
you right, you right